ls_pre_commit_hooks/openapi-linter.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+set -e
+
+LINTER_TAG="${OPENAPI_LINTER_VERSION:-21-73a3f80}"
+LINTER_IMAGE="809245501444.dkr.ecr.us-east-1.amazonaws.com/release/internal/image/openapi-linter"
+
+container_exists() {
+	if [ "$(docker ps -a -f name="$1" | wc -l)" -eq 2 ]; then
+		return 0
+	else
+		return 1
+	fi
+}
+
+run_circle_ci() {
```

Comment:
Can you add a comment with why we do this and a link to wherever you took this from? A forum post or some stackoverflow answer talking about this, etc.